HlibSokolovskyi/HW_7/7_3_hw.py

```diff
@@ -0,0 +1,13 @@
+given_string = str(input("Enter some text: \n"))
```

Comment:
look at this implementation:
def count_symbols_in_row(word):
    
    result = {}
    
    for item in word:
        if item in result:
            continue
        else:
            result.update({str(item): word.count(item)})
    return result